packages/hoc/src/withWhiteSpace/index.js

```diff
@@ -2,15 +2,15 @@ import PropTypes from 'prop-types';
 import styled from 'react-emotion';
 import { SPACING_MAP, SPACING_MAP_INDEX, MEDIA_QUERIES } from '@govuk-react/constants';
 
-const withWhiteSpace = config => (Component) => {
-  const StyledHoc = styled(Component)(({
-    mb: marginBottom = config.marginBottom,
-  }) => ({
-    marginBottom: marginBottom ? SPACING_MAP[marginBottom].mobile : 0,
-    [MEDIA_QUERIES.LARGESCREEN]: {
-      marginBottom: marginBottom ? SPACING_MAP[marginBottom].tablet : 0,
-    },
-  }));
+const withWhiteSpace = (config = {}) => (Component) => {
+  const StyledHoc = styled(Component)(({ mb: marginBottom = config.marginBottom }) => (
+    marginBottom !== undefined && {
```

Comment:
i think this is one of the cases where i'd be happier to see this as a ternery - like we had it yesterday :grin:

i.e. `marginBottom !== undefined ? { <styles here> } : undefined`